tests/config_test.py

```diff
@@ -392,6 +395,165 @@ def teardown_class(cls):
         dbconnector.load_namespace_config()
 
 
+class TestBMPConfig(object):
+    @classmethod
+    def setup_class(cls):
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        print("SETUP")
+        import config.main
+        importlib.reload(config.main)
+
+    def test_enable_bgp_neighbor_table(
+            self,
+            get_cmd_module,
+            setup_single_broadcom_asic):
+        (config, show) = get_cmd_module
+        jsonfile_config = os.path.join(mock_bmp_db_path, "bmp_invalid.json")
+        from .mock_tables import dbconnector
```

Comment:
The mock data been load to this dbconnector, but clirunner not using this.

You need load json to the config. you can check if this can work:

            # create object
            config.INIT_CFG_FILE = jsonfile_init_cfg
            config.DEFAULT_CONFIG_DB_FILE =  jsonfile_config

Reply:
Figure out issue is related to parameter @clicommon.pass_db